### PR TITLE
add definition of practice lead responsibilities

### PR DESCRIPTION
**Pull request by mattcasey**

### CODE REVIEW

File: Practices/README.md [hunk 1]
```diff
@@ -20,5 +20,19 @@ Practices
...
+* Devotes a meaningful portion (a day or more) of their week to supporting the practice.
+* Handles administrative tasks such as scheduling regular practice meetings
+* Helps the practice make decisions, via means like RFCs and group discussion
+* Ensures that all the components owned by the practice are properly maintained and monitored
+* Provides or delegates someone to help with consultation for questions that fall under the scope of the practice
+* Is a volunteer position, with the member in the lead role changing as often as quarterly, based on discussion within the practice  
+
+A lead is not:
+* Someone that people in the practice report to
+* The only leader in a practice. Core members all make decisions and promote best practices, but a lead also takes on additional responsibility for the practice fulfilling its charter.
```
Comment (caente) [inline Practices/README.md]: is it safe to say that the practice lead is not: the decider of the practice?

**Comment by caente** [comment]
@whharris indeed, that was my interpretation as well

File: Practices/README.md (hunk 1 above)
Comment (mattcasey) [inline Practices/README.md]: to understand what you mean, would an alternative name be the "decision-maker" ?

Comment (caente) [inline Practices/README.md]: I'm sure within different practices actual leaders will emerge, they will be asked for counsel and will be a tie breaker. I just want to make sure that the company is not appointing those people. And it seems we are on the same page.

Comment (whharris) [inline Practices/README.md]: Why do we need tie-breakers? I've rarely seen the need, and we don't normally have majority-rules kinds of decisions? E.g., if I am a Mongo expert commenting on a RFC regarding Mongo administration, then my comment almost automatically carries more weight than my non-expert colleagues.

**Pull request merged**